build/ghost-buster/Dockerfile

```diff
@@ -1,44 +1,57 @@
-FROM node:0.12
+FROM alpine:latest
```

Comment:
Why does this file use latest image and nginx specific version ?
Should them be the same ?